modules/ROOT/pages/configure-policy-template-source.adoc

```diff
@@ -1,31 +1,35 @@
-= Configuring Template Source for Policies in Runtime
+= Configuring Template Download Location for Policies
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: policy, CloudFront
 
 
-When an application with a tracked flow is deployed to Mule runtime engine (Mule), the API gateway of that Mule environment communicates with API Manager to request for the applied policies.
+[PA: Nahuel what is the relationship between policy and templates and how are they used. We need to set some context as to why these templates are needed. What are these templates and how are they used, why are they needed?]
 
-If the templates for the policies that are to be applied are not already available in Mule, API gateway requests API Manager for the URLs from where to download the template. When you request the template from API Manager, you must specify wether the template source is to be retrieved from a legacy system or a new one.
+When an application with a tracked API is deployed to a runtime with API gateway capabilities enabled, the runtime requests API Manager for the corresponding policies to be applied to the API.
 
+*Note:* API gateway capabilities are enabled using the organization or environment credentials where the API instance is defined.
```

Comment:
*G*ateway

Reply:
No, the way it is referred to as in docs and marketing was us to document it is is API gateway.